core_api/src/semantic_routes.py

```diff
@@ -147,7 +147,19 @@ def get_routable_chains(
             ChatRoute.ability: build_static_response_chain(ABILITY_RESPONSE, ChatRoute.ability),
             ChatRoute.coach: build_static_response_chain(COACH_RESPONSE, ChatRoute.coach),
             ChatRoute.gratitude: build_static_response_chain("You're welcome!", ChatRoute.gratitude),
-            ChatRoute.retrieval: retrieval_chain,
-            ChatRoute.summarisation: summary_chain,
+            ChatRoute.search: retrieval_chain,
```

Comment:
These changes will need to be reflected in the Django app, in [the `ChatRoute` enum](https://github.com/i-dot-ai/redbox-copilot/blob/main/django_app/redbox_app/redbox_core/models.py#L232).

Reply:
We'll rue this one day, you mark my words. But if you must, you must.

Reply:
Looks like the migration has been put in to handle route as a string in Django? Is this thread resolved?